onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -1648,6 +1649,24 @@ TEST_F(GraphTransformationTests, ReshapeFusionDistilBertTest) {
   }
 }
 
+// Test eliminating redundant Concat-Slice pattern.
+TEST_F(GraphTransformationTests, ConcatSliceEliminationTest) {
+  auto model_uri = MODEL_FOLDER "concat_slice_basic_test.onnx";
+  std::shared_ptr<Model> p_model;
+  ASSERT_STATUS_OK(Model::Load(model_uri, p_model, nullptr, *logger_));
+  Graph& graph = p_model->MainGraph();
+
+  onnxruntime::GraphTransformerManager graph_transformation_mgr{5};
+  graph_transformation_mgr.Register(onnxruntime::make_unique<ConcatSliceElimination>(), TransformerLevel::Level1);
+  auto ret = graph_transformation_mgr.ApplyTransformers(graph, TransformerLevel::Level1, *logger_);
+  ASSERT_TRUE(ret.IsOK());
+
+  Model::Save(*p_model, "concat_slice_transformed.onnx");
```

Comment:
is the model save still needed? this was for debugging purpose right?

Reply:
that's right, removing